include/memcpysampler.hpp

```diff
@@ -108,6 +108,7 @@ class MemcpySampler {
     if (unlikely(sampleMemop)) {
       writeCount();
       _memcpyTriggered++;
+      _memcpyOps = 0;
```

Comment:
Do we need this anymore? I think it's obsoleted by using the sampler.

Reply:
memcpyOps is part of the payload written to the file, and seems to be used by the profiler

Reply:
Just read it more carefully and you are correct! Thanks.